c2corg_ui/format/img.py

```diff
@@ -15,28 +15,28 @@
 
 
 class C2CImageExtension(Extension):
-
```

Comment:
Why removing this blank line? I think in all class definitions we have 1 blank line before every method. I am a bit surprised the flake8 makefile target does not raise an error for this.